src/vehicle.cpp

```diff
@@ -1103,23 +1103,23 @@ bool vehicle::start_engine( const int e )
     }
 
     if( eng.faults().count( fault_immobiliser ) ) {
-        add_msg( _( "The %s cannot start with a faulty immobiliser." ), eng.name().c_str() );
+        add_msg( _( "The %s makes a long beeping sound." ), eng.name().c_str() );
         return false;
     }
 
     if( eng.faults().count( fault_starter ) ) {
-        add_msg( _( "The %s cannot start with a faulty starter motor." ), eng.name().c_str() );
+        add_msg( _( "The %s makes a single clicking sound." ), eng.name().c_str() );
         return false;
     }
 
     int joules = eng.base.engine_start_energy( g->temperature );
     if( fuel_left( fuel_type_battery, true, true ) < joules ) {
-        add_msg( _( "The %s need at least %i battery charges to start" ), eng.name().c_str(), joules );
+        add_msg( _( "The %s makes a rapid clicking sound." ), eng.name().c_str(), joules );
```

Comment:
1. Does it? It's completely contrary to reality so how is the player supposed to determine the problem. Worse still the message will be inconsistently displayed as temperature varies. Two separate fault conditions are being confounded there so you'd need to split the conditional.

2. Previously you got a message stating what was wrong and what you needed to and as the engine got more damaged or the weather worsened it showed that you need more battery power. That's reasonably intuitive. Now it's impossible to determine what's going on without reading the source.